demo/src/app/app.component.ts

```diff
@@ -90,15 +89,17 @@ export class AppComponent {
   }
 
   initObservable() {
-    const obs = FAKE_FACT.initObservable();
     this.obsVal = 'INITIALIZED';
-    this.observableItem = obs
-      .forEach((val) => {
```

Comment:
This method worked because the `forEach` method [returns a Promise](http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-forEach).

`public forEach(next: Function, PromiseCtor: PromiseConstructor): Promise`